corehq/apps/es/aggregations.py

```diff
@@ -455,12 +455,29 @@ def __init__(self, name, datefield, interval, timezone=None):
             self.body['time_zone'] = timezone
 
 
+class NestedAggregation(Aggregation):
+    """
+    A special single bucket aggregation that enables aggregating nested documents.
+
+    :param path: Path to nested document
+    """
+    type = "nested"
+    result_class = FilterResult
+
+    def __init__(self, name, path):
+        self.name = name
+        self.body = {
+            "path": path
+        }
+
+
 AggregationTerm = namedtuple('AggregationTerm', ['name', 'field'])
 
 
-class NestedAggregationHelper(object):
+class NestedTermAggregationsHelper(object):
     """
-    Helper to run nested term-based queries (equivalent to SQL group-by clauses)
+    Helper to run nested term-based queries (equivalent to SQL group-by clauses).
+    This is not at all related to the ES 'nested aggregation'.
```

Comment:
I spend a while trying to come up with a better name but couldn't think of much else. `SubAggregationHelper` just didn't seem right.